central/globaldb/handlers/http_handler.go

```diff
@@ -57,6 +57,13 @@ func deferredRestart(ctx context.Context) {
 func RestoreDB(boltDB *bolt.DB, rocksDB *rocksdb.RocksDB) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		log.Info("Starting DB restore ...")
+		
+		// This is the old v1 API.  No need to support that for Postgres
```

Comment:
This is an assumption on my part that we don't need to support this API with Postgres.  With roxctl, it doesn't even look like there is a way to get in here without going through lots of hoops.  Though I may be missing something.

Reply:
@connorgorman What do you think? Do we need to support a case when user has old backup and want to restore it to newer version?

Reply:
Based on my limited knowledge I would think something like that would have to go through a migration vs a restore.  But I may not have the whole picture.

Reply:
well there really isn't a path in there from our code base.  roxctl uses the v2 api.  Even the stuff within `proto/api/v1/db_service.proto` goes through the v2 api via the v2_restorer and such.  The v2 restore implementation seems much more feature rich, as well.  So I figured that api was already on the way out and a major upgrade such as this would make a good time to pull the plug.  

But of course I called it out here as I was not 100% certain of that desire.  I can support it if we need to easy enough, but I think one path for Postgres restore code is better, if possible.

Reply:
Yeah I think we can support the v2 api only moving forward and using roxctl as the only supported way to restore

Reply:
How are we handling that with Postgres?  It was essentially deprecated with version 27.0.  But as far as these changes are concerned, we won't support this API for Postgres.  So how would we do the CHANGE.log for that since while the code will be in the next release, it wouldn't impact anyone until Postgres was enabled.  Are we capturing such things elsewhere so we can dump them all in when Postgres goes?  I don't want to confuse the CHANGE.log with things the users won't see or know nothing about.

Reply:
I would say in the CHANGELOG that when Postgres is released, the v1 API will become unsupported